ietf-te-types.yang

```diff
@@ -2111,89 +2198,231 @@ module ietf-te-types {
-  }
-
-  identity path-metric-te {
-    base path-metric-type;
-    description
-      "TE path metric.";
-    reference
-      "RFC 3785: Use of Interior Gateway Protocol (IGP) Metric as a
-       second MPLS Traffic Engineering (TE) Metric";
-  }
-
-  identity path-metric-igp {
-    base path-metric-type;
+  identity path-metric-optimization-type {
     description
-      "IGP path metric.";
-    reference
-      "RFC 3785: Use of Interior Gateway Protocol (IGP) Metric as a
-       second MPLS Traffic Engineering (TE) Metric";
+      "Base identity used to define the path metric optimization
+      types.";
   }
 
-  // CHANGE NOTE: The reference for the identity path-metric-hop
+  // CHANGE NOTE: The link-path-metric-type base identity
   // has been added in this module revision
   // RFC Editor: remove the note above and this note
-  identity path-metric-hop {
-    base path-metric-type;
+  identity link-path-metric-type {
     description
-      "Hop path metric.";
-    reference
-      "RFC5440: Path Computation Element (PCE) Communication
-      Protocol (PCEP)";
+      "Base identity used to define the link and the path metric
+      types.
+      
+      The unit of the path metric value is interpreted in the
+      context of the path metric type and the derived identities
+      SHOULD describe the unit of the path metric types they
+      define.";
   }
 
+    // CHANGE NOTE: The link-metric-type base identity
+    // and its derived identities
+    // have been added in this module revision
+    // RFC Editor: remove the note above and this note
+    identity link-metric-type {
+      base link-path-metric-type;
+      description
+        "Base identity for the link metric types.";
+    }
 
-  identity path-metric-delay-average {
-    base path-metric-type;
-    description
-      "Average unidirectional link delay.";
-    reference
-      "RFC 7471: OSPF Traffic Engineering (TE) Metric Extensions";
-  }
+      identity link-metric-te {
+        base link-metric-type;
+        description
+          "Traffic Engineering (TE) Link Metric.";
+        reference
+          "RFC 3630: Traffic Engineering (TE) Extensions to OSPF
+          Version 2, section 2.5.5
 
-  identity path-metric-delay-minimum {
-    base path-metric-type;
-    description
-      "Minimum unidirectional link delay.";
-    reference
-      "RFC 7471: OSPF Traffic Engineering (TE) Metric Extensions";
-  }
+          RFC 5305: IS-IS Extensions for Traffic Engineering,
+          section 3.7";
+      }
 
-  identity path-metric-residual-bandwidth {
-    base path-metric-type;
-    description
-      "Unidirectional Residual Bandwidth, which is defined to be
-       Maximum Bandwidth (RFC 3630) minus the bandwidth currently
-       allocated to LSPs.";
-    reference
-      "RFC 3630: Traffic Engineering (TE) Extensions to OSPF
-       Version 2
-       RFC 7471: OSPF Traffic Engineering (TE) Metric Extensions";
-  }
+      identity link-metric-igp {
```

Comment:
can you check if right alignment (4 spaces?)

Reply:
See: https://github.com/tsaad-dev/te/pull/262/files#r1498249288